index.html

```diff
@@ -73,9 +73,9 @@ <h2>Supporters</h2>
               <img src="images/supporters/redhat.svg" height="50px" width="auto" alt="RedHat Logo"/>
               <a title="Stickermule" href="http://stickermule.com/supports/opensourcedesign19-sponsorship" target="_blank" rel="noopener"><img src="images/supporters/stickermule.svg" height="40px" width="auto" alt="Stickermule Logo"/></a>
               <img src="images/supporters/wikimedia-germany.svg" height="50px" width="auto" alt="Wikimedia Germany Logo"/>
+              <center><object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/sponsor.svg?avatarHeight=50&width=110"></object></center>
```

Comment:
@Yashank18 `<center> .. </center>` tag is deprecated now. I encourage you not to use this.

Please **play around CSS to center align this.** One method of centering is to set the `margin-left` and `margin-right` properties of the element to `auto`, and then set the parent element’s `text-align` property to center. This guarantees that the element will be centered in all modern browsers.

Reply:
@vinitshahdeo Thank you for reviewing. It was my bad. I just realized its parent element already has a center align in CSS so I just made changes, These will solve the issue and it increases the size, as well as this button, wasn't that noticeable.